content/metadata.yaml

```diff
@@ -514,6 +514,9 @@ authors:
     contributions:
       - Writing - Original Draft
       - Writing - Review & Editing
+    affiliations:
+      - freelance copyeditor
```

Comment:
Affiliations need location (city, country)

Reply:
-the author who has an open source affiliation listed did not put city, country because that's an example of stuff that get's done online from anywhere. (& obviously, open source is an idea that's a big part of CS)
-freelance copyediting can also be done from anywhere (& b/c I've done it since grade school then I've done it in multiple places -- it's not associated with a location, it's associated with me -- and if I imply that it's associated with a location then that loses the flexibility which allows it to apply so widely [which would be ridiculous because why would I diminish work during a pandemic?]). there's lots of work like this and now with the pandemic more people are aware of it and developing such options for wherever they've managed to hunker down for the pandemic, but it was always around -- if we didn't have these options, there'd be a lot less work getting done now. With what I have written I think it's sufficient.
-some people wrote multiple locations even though they're clearly not simultaneously there. I can't add a "location" to the first affiliation without losing the flexibility inherent in & necessary for that affiliation (the most I can do is add "USA-based"), so if you have to have a city, country then I can just put the obvious for the 2nd recent PhD affiliation. I'll update it myself so you can approve it.
thanks!